platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/crypto/CryptoArgsProvider.java

```diff
@@ -53,9 +54,14 @@ static Stream<Arguments> basicTestArgs() throws Exception {
         final Map<NodeId, KeysAndCerts> genC = CryptoStatic.generateKeysAndCerts(genAB);
         System.out.println(
                 "Key generating took " + Duration.between(start, Instant.now()).toMillis());
+        start = Instant.now();
+        final AddressBookAndCerts addressBookAndCerts2 = createAddressBookLoadSigKeysCreateAgrKeys(NUMBER_OF_ADDRESSES);
+        System.out.println("Agreement key generating took "
+                + Duration.between(start, Instant.now()).toMillis());
         return Stream.of(
                 Arguments.of(addressBookAndCerts.addressBook(), addressBookAndCerts.nodeIdKeysAndCertsMap()),
-                Arguments.of(genAB, genC));
+                Arguments.of(genAB, genC),
+                Arguments.of(addressBookAndCerts2.addressBook(), addressBookAndCerts2.nodeIdKeysAndCertsMap()));
```

Comment:
This addition to the output stream isn't reflected in the `@return` javadoc

Reply:
added comments reflecting the additional argument.